dipy/align/bundlemin.pyx

```diff
@@ -242,6 +242,99 @@ def _bundle_minimum_distance(double [:, ::1] stat,
     return dist
 
 
+
+def _bundle_minimum_distance_asymmetric(double [:, ::1] static,
+                                        double [:, ::1] moving,
+                                        cnp.npy_intp static_size,
+                                        cnp.npy_intp moving_size,
+                                        cnp.npy_intp rows):
+    """ MDF-based pairwise distance optimization function
+
+    We minimize the distance between moving streamlines of the same number of
+    points as they align with the static streamlines.
+
+    Parameters
+    -----------
+    static : array
+        Static streamlines
+    moving : array
+        Moving streamlines
+    static_size : int
+        Number of static streamlines
+    moving_size : int
+        Number of moving streamlines
+    rows : int
+        Number of points per streamline
+
+    Returns
+    -------
+    cost : double
+
+    Notes
+    -----
+    The difference with ``_bundle_minimum_distance`` is that we sum the
+    minimum values only for the static. Therefore, this is asymetric.
```

Comment:
Can you please explain more what are the consequences, if any? This function is a little less heavy than `_bundle_minimum_distance`, ok, but what are we losing? When can we use it?